### ISSUE TITLE
Accessibility: aria-label for Form.Check?

### ISSUE THREAD

**Issue by JeffStodd** [react-bootstrap/react-bootstrap#5598]
As you can see these sections are blank, I would like to apply a label to the empty fields
![](https://i.imgur.com/que05Ch.png)

Here is my source code
![](https://i.imgur.com/Mzqf21b.png)

The HTML is generated like this, and I would like to apply an aria-label to this div tag:
`<div class="custom-control custom-switch">
    <input required="" type="checkbox" id="Fever" class="custom-control-input">
    <label title="" for="Fever" class="custom-control-label">Fever</label>
</div>`

Problem is, the source for react bootstrap generates the html without any label options
`<div
          style={style}
          className={classNames(
            className,
            bsPrefix,
            custom && `custom-${type}`,
            inline && `${bsPrefix}-inline`,
          )}
        >
        ...
        children
        ...
</div>`

**Comment by jquense** [comment]
Hey there, not sure i understand why you'd want to do this, the FormCheck already gets its label from the `<label>` it's wrapped in, and my understanding is that adding a aria-label is at least unnecessary or maybe even wrong.

**Comment by JeffStodd** [comment]
> Hey there, not sure i understand why you'd want to do this, the FormCheck already gets its label from the `<label>` it's wrapped in, and my understanding is that adding a aria-label is at least unnecessary or maybe even wrong.

I am new to the accessibility stuff so I apologize if anything I am doing is out of spec.
By my understanding and according to the Firefox accessibility web developer tool, screen readers will read through each section as the user focuses/selects them, and the user controls if they would like to further focus/read any nested sections, or skip to the next one. When the user goes over the `<div class="custom-control custom-switch">` section, as highlighted in the first picture, the screen reader "sees" an empty string. They would have to explore the nested section to discover that it is a checkbox labeled "x", since the `<label>` attribute only applies to the nested `<input>`, which is not very helpful.

Basically, I'm asking if it's possible to label the generated `<div>` wrapper through the JSX `<Form.Check>`.
